src/main/kotlin/com/openlattice/search/renderers/BHRAlertEmailRenderer.kt

```diff
@@ -116,10 +120,11 @@ class BHRAlertEmailRenderer {
                 persistentSearch: PersistentSearch,
                 report: Map<FullQualifiedName, Set<Any>>,
                 userEmail: String,
-                neighbors: List<NeighborEntityDetails>
+                neighbors: List<NeighborEntityDetails>,
+                alertName: String
         ): RenderableEmailRequest {
 
-            val subject = "New Behavioral Health Report"
+            val subject = "New Crisis Report ($alertName)"
```

Comment:
I think it would be a bit cleaner to save `alertName` on the `alertMetadata` and look it up from there instead of adding to the enum -- this will also make it much easier to add additional BHR alert labels in the future (especially important if they might vary by organization).

i.e. when you're creating the persistent search on the frontend:
```
const persistentSearch = {
    type: 'BHR_ALERT',
    ...,
    alertMetadata: {
        alertName: 'University of Iowa Affiliate' // or Veteran, or whatever
    },
    ...
}
```
and then in `BHRAlertEmailRenderer` at the top under `/* METADATA TAGS */`:
```
const val ALERT_NAME = "alertName"
```
and in `renderEmail`:
```
val alertName = persistentSearch.alertMetadata[ALERT_NAME]?.let { " ($it)" } ?: ""
val subject = "New Crisis Report$alertName"
```

Reply:
`...?.let { " ($it)" } ?: ""`
This is quite the syntax...